doc-content/enterprise-only/installation/eap-data-source-add-proc.adoc

```diff
@@ -10,7 +10,7 @@ For production environments, specify an actual data source. Do not use the examp
 
 .Prerequisites
 * The JDBC providers that you want to use to create database connections are configured on all servers on which you want to deploy {KIE_SERVER}, as described in the "Creating Datasources" and "JDBC Drivers" sections of the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/index[_Red Hat JBoss Enterprise Application Server Configuration Guide_].
-* The *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`) file has been downloaded, as described in <<install-download-proc_{context}>>.
+* The {PRODUCT} {PRODUCT_VERSION_LONG} Add Ons ({PRODUCT_FILE}-add-ons.zip) file has been downloaded from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal.
```

Comment:
Remove "has been". See style guide for more information on tense.

Reply:
Done